lib/Field/FieldRepeatButton.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
-import { bool } from 'prop-types';
+import { bool, func } from 'prop-types';
 import { ButtonSecondary } from '../ButtonNew/ButtonSecondary/ButtonSecondary';
 import Icon from '../Icon';
 import { FieldMainColumn } from './FieldMainColumn';
 
-export function FieldRepeatButton({ repeatLimitReached }) {
+export function FieldRepeatButton({ repeatLimitReached, repeatField }) {
```

Comment:
🔴 is there a reason why we didn't use onClick instead of repeatField? I would prefer onClick here because it's a common pattern/name & it's easier to understand.

Reply:
Ah ok, I think it was because I'd added the other handlers (trash, move up, move down) explicitly on the RepeatControls and I just added this at the same time, but i like onClick here also.